libfreerdp/core/nego.c

```diff
@@ -57,7 +58,7 @@ struct rdp_nego
 	UINT32 SelectedProtocol;
 	UINT32 RequestedProtocols;
 	BOOL NegotiateSecurityLayer;
-	BOOL EnabledProtocols[16];
+	BOOL EnabledProtocols[32];
```

Comment:
why did this get bumped to 32? just curious as 16 looks safe enough, but I don't mind extra caution

Reply:
`#define PROTOCOL_RDSAAD 0x00000010` might explain it ;)